network/p2p/test_utils.go

```diff
@@ -147,20 +158,31 @@ func (ln *LocalNet) NewTestP2pNetwork(ctx context.Context, nodeIndex int, keys t
 	dutyStore := dutystore.New()
 	signatureVerifier := signatureverifier.NewSignatureVerifier(nodeStorage)
 
-	cfg := NewNetConfig(keys, format.OperatorID(operatorPubkey), ln.Bootnode, testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), options.Nodes)
+	cfg := NewNetConfig(keys, format.OperatorID(operatorPubkey), ln.Bootnode, ssv_testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), options.Nodes)
 	cfg.Ctx = ctx
 	cfg.Subnets = "00000000000000000000020000000000" //PAY ATTENTION for future test scenarios which use more than one eth-validator we need to make this field dynamically changing
-	cfg.NodeStorage = mock.NodeStorage{
-		MockPrivateKeyHash:              hash,
-		RegisteredOperatorPublicKeyPEMs: []string{},
+	if err != nil {
```

Comment:
why are you checking err here? it hasn't changed since L154

Reply:
leftover, removed